src/main/java/module/series/Model.java

```diff
@@ -1,14 +1,44 @@
 package module.series;
 
+import core.db.DBManager;
+import core.model.enums.MatchType;
 import core.model.enums.RatingsStatistics;
 
+import java.util.HashMap;
 import java.util.Map;
 
 public class Model {
 
+	public static class StatisticsEntry {
+		private int count;
+		private int sum;
+		private int max;
+
+		public StatisticsEntry(int v){
+			count=1;
+			sum=v;
+			max=v;
+		}
+
+		public void addValue(int v) {
+			count++;
+			sum += v;
+			if (v > max) max = v;
+		}
+
+		public int getMax() {
+			return max;
+		}
+
+		public int getAverage() {
+			if (count > 0) return (int) ((double) sum / count + .5);
```

Comment:
What is the `+ .5` for here?  looks hackish ;-)

Reply:
this is how the c++ developer rounds off

Reply:
changed to Math.round